src/tox/config/__init__.py

```diff
@@ -595,7 +595,7 @@ def tox_addoption(parser):
     )
 
     def _set_envdir_from_devenv(testenv_config, value):
-        if testenv_config.config.option.devenv is not None:
+        if testenv_config.config.option.devenv is not None and testenv_config.envname != ".tox":
```

Comment:
This cannot be a static value, you need to read it from https://tox.readthedocs.io/en/latest/config.html#conf-provision_tox_env

Reply:
Thank you for the pointer! That has been fixed